src/Release/GetReleaseToggles/ResponseToggle.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Clearbooks\Labs\Release\GetReleaseToggles;
+
+class ResponseToggle
```

Comment:
It's called ResponseToggle cause getTogglesForRelease returns an array of ResponseToggles, it hides the Toggle instance.

I'd prefer if GetReleaseToggles->execute would return a TogglesOfRelease response class containing an array of `Toggle`. What do you think @adrianclay, @viktorcb  ?

I think creating a ResponseToggle class just would make this unnecessarily cut up into small pieces. And it doesn't really give any extensionability cause in the end you'll want all toggle information anyway.